src/main/java/com/questhelper/helpers/quests/atfirstlight/AtFirstLight.java

```diff
@@ -0,0 +1,356 @@
+		hammer = new ItemRequirement("Hammer", ItemCollections.HAMMER).isNotConsumed();
+		hammer.canBeObtainedDuringQuest();
+		jerboaTail = new ItemRequirement("Jerboa tail", ItemID.JERBOA_TAIL);
+		jerboaTail.canBeObtainedDuringQuest();
+
+		jerboaTailOrBoxTrap = new ItemRequirements(LogicType.OR, "Jerboa tail, or a box trap to get some", jerboaTail, boxTrap);
+		jerboaTail2OrBoxTrap = new ItemRequirements(LogicType.OR, "2 Jerboa tails, or a box trap to get some", jerboaTail.quantity(2), boxTrap);
+
+		// Recommended
+		staminaPotion = new ItemRequirement("Stamina potions", ItemCollections.STAMINA_POTIONS);
+
+		// Quest Items
+		toyMouse = new ItemRequirement("Toy mouse", ItemID.TOY_MOUSE);
+		toyMouseWound = new ItemRequirement("Toy mouse (wound)", ItemID.TOY_MOUSE_WOUND);
+		smoothLeaf = new ItemRequirement("Smooth leaf", ItemID.SMOOTH_LEAF);
+		stickyLeaf = new ItemRequirement("Sticky leaf", ItemID.STICKY_LEAF);
+		makeshiftPoultice = new ItemRequirement("Makeshift poultice", ItemID.MAKESHIFT_POULTICE);
+		furSample = new ItemRequirement("Fur sample", ItemID.FUR_SAMPLE);
+		trimmedFur = new ItemRequirement("Trimmed fur", ItemID.TRIMMED_FUR);
+	}
+
+	private void setupConditions()
+	{
+		inGuild = new ZoneRequirement(guild);
+		gotMouse = new VarbitRequirement(9843, 1);
+		usedMouse = new VarbitRequirement(9839, 1);
+		checkedBed = new VarbitRequirement(9837, 1);
+
+		// 9842 0->1, received pelt once
+		equipmentUsable = new VarbitRequirement(9840, 1);
+		repairedEquipment = new VarbitRequirement(9840, 2);
+		handedInReport = new VarbitRequirement(9836, 1);
+		hadReport = or(foxsReport, handedInReport);
+
+		foxsReport = new ItemRequirement("Fox's report", ItemID.FOXS_REPORT).hideConditioned(handedInReport);
+
+		// Bed repaired, 9838 0->1
+	}
+
+	private void setupSteps()
+	{
+		talkToApatura = new NpcStep(this, NpcID.GUILDMASTER_APATURA, new WorldPoint(1554, 3033, 0),
+			"Talk to Guildmaster Apatura in the Hunter Guild, south-west of Civitas illa Fortis.");
+		talkToApatura.addDialogSteps("Can I help you with anything?", "Yes.");
+		goDownTree = new ObjectStep(this, ObjectID.STAIRS_51641, new WorldPoint(1557, 3048, 0),
+			"Go down the stairs in the tree in the guild.");
+		talkToVerity = new NpcStep(this, NpcID.GUILD_SCRIBE_VERITY, new WorldPoint(1559, 9464, 0),
+			"Talk to Guild Scribe Verity behind the bar.");
+		talkToWolf = new NpcStep(this, NpcID.GUILD_HUNTER_WOLF_MASTER, new WorldPoint(1555, 9462, 0),
+			"Talk to Guild Hunter Wolf (Master), next to the bar.");
+		windUpToy = new DetailedQuestStep(this, "Wind up a toy mouse.", toyMouse.highlighted());
+		windUpToy.addIcon(ItemID.TOY_MOUSE_WOUND);
+		useToyOnKiko = new NpcStep(this, NpcID.GUILD_HUNTER_KIKO, new WorldPoint(1552, 9460, 0),
+			"Use the wound up toy mouse on Guild Hunter Kiko near the bar.", toyMouseWound.highlighted());
+		checkBed = new ObjectStep(this, ObjectID.CAT_BED, new WorldPoint(1552, 9460, 0),
+			"Search the cat's bed.");
```

Comment:
nit: Technically we're checking the bed, not searching it. Very unimportant change - worst case the cat gets an extra pet
![check-bed](https://github.com/Zoinkwiz/quest-helper/assets/962989/486a5207-7286-45e8-9dd0-24eb8280b780)

```suggestion
			"Check the cat's bed.");
```
or
```suggestion
			"Right-click check the cat's bed.");
```
or
```suggestion
			"Right-click the cat's bed and check it.");
```